StreamChatClient_v3/Database/DTOs/ChannelDTO_Tests.swift

```diff
@@ -116,6 +116,44 @@ class ChannelModelDTO_Tests: XCTestCase {
         }
     }
     
+    func test_channelListQuery_withSorting() {
+        let filter = Filter.equal("some", to: String.unique)
```

Comment:
Please add human-readable comments to tests so it's easier to understand what's going on. For example:
```
// Create two channels queries with different sortings
```